packages/client/src/forms/index.ts

```diff
@@ -1257,3 +1257,37 @@ export interface ICertificate {
   payments?: Payment[]
   data?: string
 }
+
+export function modifyFormField(
```

Comment:
Related to a small refactor I made [here](https://github.com/opencrvs/opencrvs-core/pull/7301/files#diff-a130959cffa05c698bc62a241f9098ff13ccd5c2e0dfd477c9d2a5602c42d29eR384) while debugging something